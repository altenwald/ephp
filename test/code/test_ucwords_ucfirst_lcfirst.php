<?php
$foo = 'hello world!';
$foo = ucwords($foo);             // Hello World!
var_dump($foo);

$bar = 'HELLO WORLD!';
$bar = ucwords($bar);             // HELLO WORLD!
var_dump($bar);
$bar = ucwords(strtolower($bar)); // Hello World!
var_dump($bar);

$foo = 'hello|world!';
$bar = ucwords($foo);             // Hello|world!
var_dump($bar);

$baz = ucwords($foo, "|");        // Hello|World!
var_dump($baz);

$foo = 'hello world!';
$foo = ucfirst($foo);             // Hello world!
var_dump($foo);

$bar = 'HELLO WORLD!';
$bar = ucfirst($bar);             // HELLO WORLD!
var_dump($bar);
$bar = ucfirst(strtolower($bar)); // Hello world!
var_dump($bar);

$foo = 'HelloWorld';
$foo = lcfirst($foo);             // helloWorld
var_dump($foo);

$bar = 'HELLO WORLD!';
$bar = lcfirst($bar);             // hELLO WORLD!
var_dump($bar);
$bar = lcfirst(strtoupper($bar)); // hELLO WORLD!
var_dump($bar);
