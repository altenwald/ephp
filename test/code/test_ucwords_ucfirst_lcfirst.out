string(12) "Hello World!"
string(12) "HELLO WORLD!"
string(12) "Hello World!"
string(12) "Hello|world!"
string(12) "Hello|World!"
string(12) "Hello world!"
string(12) "HELLO WORLD!"
string(12) "Hello world!"
string(10) "helloWorld"
string(12) "hELLO WORLD!"
string(12) "hELLO WORLD!"
